src/index.html

```diff
@@ -17,6 +20,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
BEM naming convention suggests that modifiers should be used with the block or element name. If 'stopwatch--speed-up' is a modifier, it should be used alongside the base class 'stopwatch'.